Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -426,11 +419,18 @@ public class CertificateValidationRequest : CommonRequest {
                     return
                 }
                 
+                let regexp = "[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}"
```

Comment:
Looks like the UID regexp is used in a few places. Might be good to just have this in our `Constants` file. (Mostly noting for myself, unless you feel ambitious)